include/z64.h

```diff
@@ -184,10 +184,22 @@ typedef struct {
     /* 0x0104 */ Vec3f  curDistortionOrientation;
     /* 0x0110 */ Vec3f  curDistortionScale;
     /* 0x011C */ u16    normal; // used to normalize the projection matrix
-    /* 0x0120 */ s32    flags;
+    /* 0x0120 */ s32    dirty;
```

Comment:
`dirtyFlags`? Might feel redundant but I think it's nice to still indicate it's a set of flags.

Reply:
Nice 👍